janusgraph-dist/src/assembly/static/bin/janusgraph-server.sh

```diff
@@ -254,8 +260,18 @@ startForeground() {
   fi
 }
 
+printConfig() {
+    echo "JANUSGRAPH_HOME: ${JANUSGRAPH_HOME}"
+    echo "JANUSGRAPH_YAML: ${JANUSGRAPH_YAML}"
+    echo "JANUSGRAPH_SERVER_CMD: ${JANUSGRAPH_SERVER_CMD}"
+    echo "LOG_DIR: ${LOG_DIR}"
+    echo "PID_FILE: ${PID_FILE}"
+    echo "JAVA_OPTIONS: ${JAVA_OPTIONS}"
+    echo "COLLECTED_JAVA_OPTIONS_FILE: ${COLLECTED_JAVA_OPTIONS_FILE}"
```

Comment:
Here, I would expect to get a file path as this ends on `FILE` but I'm actually getting its content:

```
COLLECTED_JAVA_OPTIONS_FILE:  -XX:+CMSClassUnloadingEnabled
```

is that supposed to be like that? If so, then I think that the name is confusing. I'm also not sure if we really want to include this output here. Isn't that only useful to debug the problem this is supposed to fix?

Reply:
Will remove this log as they are already include in JAVA_OPTIONS